src/lib.rs

```diff
@@ -57,6 +57,12 @@ impl From<std::io::Error> for ToipeError {
     }
 }
 
+impl std::fmt::Display for ToipeError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str(self.msg.as_str())
```

Comment:
This doesn't provide any context about what kind of error it is. For example, if the `msg` is `file not found`, you would get just `file not found` in the terminal. Instead, I think it would be better to print `ToipeError: File not found`.

Reply:
done. the new error would be `Error: ToipeError: error_msg`.